Consolidate to one `schedule` param (#25410)
mistercrunch/Airflow: docs/apache-airflow/dag-run.rst

Cron Presets
''''''''''''

> You may set your DAG to run on a simple schedule by setting its ``schedule`` param with either a
> `cron expression <https://en.wikipedia.org/wiki/Cron#CRON_expression>`_, or a ``datetime.timedelta`` object,
> or one of the following cron "presets". For more elaborate scheduling requirements, you can implement a :doc:`custom timetable </concepts/timetable>`

Reviewer: ```suggestion
You may set your DAG to run on a simple schedule by setting its ``schedule`` argument to either a
`cron expression <https://en.wikipedia.org/wiki/Cron#CRON_expression>`_, a ``datetime.timedelta`` object,
or one of the following cron "presets". For more elaborate scheduling requirements, you can implement a :doc:`custom timetable </concepts/timetable>`
```

Each DAG run in Airflow has an assigned "data interval" that represents the time
range it operates in. For a DAG scheduled with ``@daily``, for example, each of

> ts data interval would start each day at midnight (00:00) and end at midnight

Reviewer 2: ```suggestion
its data interval would start each day at midnight (00:00) and end at midnight
```

.. tip::

>     If a cron expression or `timedelta` object is not enough to express your DAG's schedule,

Reviewer: ```suggestion
    If a cron expression or timedelta object is not enough to express your DAG's schedule,
```

Catchup
-------

> An Airflow DAG defined with a ``start_date``, possibly an ``end_date``, and either a cron expression or timetable, defines a series of intervals which the scheduler turns into individual DAG Runs and executes.

Reviewer: ```suggestion
An Airflow DAG defined with a ``start_date``, possibly an ``end_date``, and a non-dataset schedule, defines a series of intervals which the scheduler turns into individual DAG runs and executes.
```

> An Airflow DAG defined with a ``start_date``, possibly an ``end_date``, and a either a cron expression or timetable, defines a series of intervals which the scheduler turns into individual DAG Runs and executes.

Reviewer 2: ```suggestion
An Airflow DAG defined with a ``start_date``, possibly an ``end_date``, and either a cron expression or timetable, defines a series of intervals which the scheduler turns into individual DAG Runs and executes.
```

I think the sentence still need a little love.
